app/tests/spec/views/sign_in.js

```diff
@@ -146,6 +146,32 @@ function (chai, $, sinon, p, View, Session, AuthErrors, OAuthErrors, Metrics,
               assert.equal(view.$('[type=email]').val(), 'testuser@testuser.com');
             });
       });
+
+      it('displays a message if migration relier is set', function () {
+        sinon.stub(relier, 'has', function (arg) {
```

Comment:
Since you have tested the `isMigration` functionality above, you could event just stub in `view.isMigration` here and ignore the relier.